internal/testing/exchange/exchange_test.go

```diff
@@ -0,0 +1,35 @@
+package exchange
+
+import (
+	"testing"
+
+	"github.com/gorilla/websocket"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/config"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/binance"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+)
+
+// TestTestInstance exercises TestInstance
+func TestTestInstance(t *testing.T) {
+	b := new(binance.Binance)
+	require.NoError(t, TestInstance(b), "TestInstance must not error")
+	assert.NotNil(t, b.Websocket, "TestInstance should set up a websocket")
+
+	e := new(sharedtestvalues.CustomEx)
+	require.ErrorIs(t, TestInstance(e), config.ErrExchangeNotFound, "TestInstance error correctly on a missing exchange")
+}
+
+// TestMockHTTPInstance exercises MockHTTPInstance
+func TestMockHTTPInstance(t *testing.T) {
+	b := new(binance.Binance)
+	require.NoError(t, TestInstance(b), "TestInstance must not error")
+	require.NoError(t, MockHTTPInstance(b), "MockHTTPInstance must not error")
```

Comment:
Understood.
I definitely took your feedback on board, and I've tried to strike a balance.
I found myself writing two step quite often, and then in self-review it wouldn't taste right.
I'm still definitely using 2 step whenever it's doesn't parse very quickly or make immediate sense,
but like you say, that's subjective.
I think in this case I *want* you to skip over this boilerplate, like you say.
I don't want you to care that this test has done some scaffolding first, because it's not relevant to the substance of the test, *unless* that scaffolding goes wrong somehow.